package.json

```diff
@@ -16,6 +16,7 @@
     "@docusaurus/theme-search-algolia": "^2.0.1",
     "@mdx-js/react": "^1.5.8",
     "clsx": "^1.1.1",
+    "node": "^18.8.0",
```

Comment:
This was put here by yarn 1.22.19.

Reply:
We build using a docker image so I am not sure that this will have any effect.
Docusaurus does support node 18 so we should be OK